WordPress/gutenberg
# Add initial support for themes to control the editor

Fixes https://github.com/WordPress/gutenberg/issues/22655
Follows-up https://github.com/WordPress/gutenberg/pull/22291
Related https://github.com/WordPress/gutenberg/issues/20588

This PR adds support for themes to control the editor via the `global` context.

It also merges the content of `editor-features.php` within `global-styles.php`, improving the performance as a result of having only one filesystem access to the same file by request.

## How to test

- Load this PR.
- Load edit-site and verify that the value `settings.__experimentalFeatures.typography.dropCap` at the `block-editor` store is `false`.
- Open the file `lib/experimental-theme.json` and set the dropcap value to `true` => verify the value is `true` in the store.
- Install and enable a theme with support for global styles (for example, the [demo theme](https://github.com/nosolosw/global-styles-theme/)). Set dropcap to `false` in the theme's `experimental-theme.json` file => verify the value is false in the store.
